pubkey_encrypt.install

```diff
@@ -0,0 +1,63 @@
+<?php
+
+/**
+ * @file
+ * Install, update and uninstall functions for the pubkey_encrypt module.
+ */
+
+/**
+ * Implements hook_requirements().
+ */
+function pubkey_encrypt_requirements($phase) {
+  $requirements = array();
+  // Check for requirements during install.
+  if ($phase == 'install') {
+    // Check if openssl is enabled.
+    if (!extension_loaded('openssl')) {
```

Comment:
I think the Encrypt API doesn't provide functionality for generating Public/Private key pairs. So that's why it's here. It would be removed though after a pluggable system for Public/Private key pairs generation is added.